app/services/katello/pulp3/smart_proxy_repository.rb

```diff
@@ -63,6 +61,10 @@ def orphan_repository_versions
         repo_version_map
       end
 
+      def publication_for_repository_href(repository_version_href)
```

Comment:
Is this used anywhere?

Reply:
This was a helper for the distribution lookup I had earlier. Let me remove this.